draft-privacypass-rate-limit-tokens.md

```diff
@@ -1262,11 +1259,67 @@ unique among all currently in rotation for the origin.
 
 This document updates the "Token Type" Registry ({{AUTHSCHEME}}) with the following value:
 
-| Value  | Name                   | Publicly Verifiable | Public Metadata | Private Metadata | Nk  | Nid | Reference        |
-|:-------|:-----------------------|:--------------------|:----------------|:-----------------|:----|:----|:-----------------|
-| 0x0003 | Rate-Limited Blind RSA | Y                   | N               | N                | 512 | 32  | This document    |
+| Value  | Name                                                                 | Publicly Verifiable | Public Metadata | Private Metadata | Nk  | Nid | Reference        |
+|:-------|:---------------------------------------------------------------------|:--------------------|:----------------|:-----------------|:----|:----|:-----------------|
+| 0x0003 | Rate-Limited Blind RSA(SHA-384, 2048-bit) with ECDSA(P-384, SHA-384) | Y                   | N               | N                | 512 | 32  | This document    |
+| 0x0004 | Rate-Limited Blind RSA(SHA-384, 2048-bit) with Ed25519(SHA-512)      | Y                   | N               | N                | 512 | 32  | This document    |
```

Comment:
nit: Does the SHA-512 need to be specified here?

Reply:
Yeah, since we use it with HKDF to compute anon_origin_id.